File: src/main/java/dev/retrotv/crypto/exception/CryptoFailException.java
package dev.retrotv.crypto.exception;

/**
 * 암복호화 시, 발생할 수 있는 오류를 처리하기 위한 예외입니다.
 *
 * @author  yjj8353
 * @since   1.8
 */
public class CryptoFailException extends RuntimeException {

    public CryptoFailException() {
        super();
    }

    public CryptoFailException(String message) {
        super(message);
    }

    public CryptoFailException(String message, Throwable throwable) {
        super(message, throwable);
    }
}

File: src/main/kotlin/dev/retrotv/crypto/exception/CryptoFailException.kt
package dev.retrotv.crypto.exception

/**
 * 암복호화 시, 발생할 수 있는 예외를 처리하기 위한 예외입니다.
 *
 * @author  yjj8353
 * @since   1.0.0
 */
class CryptoFailException : RuntimeException {
    constructor() : super()
    constructor(message: String) : super(message)
    constructor(message: String, throwable: Throwable) : super(message, throwable)
}
